### DISCUSSION TITLE
外壳的成本有办法下降吗

### DISCUSSION THREAD

**Discussion by XieJinYan1127** [peng-zhihui/HelloWord-Keyboard#89] [Q&A]
#### 背景
查了淘宝的3D打印售价。
基础版的3D打印价格在400以上。
#### 讨论
1. 个人使用的情况，有更低成本低的外壳制作方案吗

**Comment by crafter-z** [comment]
自定义的结构件加工就没有便宜的，除非拼团。不论是单独开模注塑，还是CNC,还是3D打印，小批量的自定义结构件加工从来都很贵，降不下来的

**Comment by wlfinal** [comment]
3D打印主要是机时贵（因为速度太慢），电费、材料、人工都没多少的。自己有打印机最好了，找别人打小件还行，400块还不如买一台。
可以咨询一下实木的CNC价格，应该会比铝低一些（猜的），或者用海洋板、亚克力做CNC再拼接。
再便宜手工做模具，用环氧树脂灌注，手动打磨，这样应该是最低了，就是很费时间。
或者买量产的塑料键盘壳，自己改PCB就行。
时间或者钱，总要出一个。
而且说实话，就算做出来，达不到视频里的质量的，光是基础版没什么意义，你还不如买量产键盘，400也能选不少了。

**Reply by Sheihhill** [reply]
同意楼上说法，像我一样一开始只是对移位寄存器就行通信，可以全键无冲表示很喜欢，可是一套下来发现我其实只需要一个无冲的三模键盘，没有必要搞这个，所以这个项目全当看看就行，如果是想要体验到那些其他的东西，要么需要钱，要么需要能力。

**Reply by ZhangJiaLei123** [reply]
3D打印的触感和结构强度坑定不如CNC的

**Comment by ZhangJiaLei123** [comment]
抖音有人团基础版，2.5k， 完全版估计3k多。 自己做的话， 我们公司之前做了CNC的外壳，差不多尺寸10个3w

**Comment by xdeng** [comment]
同求普通塑料+无扩展版

**Comment by xdeng** [comment]
搞这么贵就不利于传承。

**Comment by Altermoe** [comment]
用木头切一个
